notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -70,340 +75,461 @@
-   "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "source": "expt_name = \"RKL4982\"",
    "outputs": [],
-   "source": [
-    "sample_accession_fp = './test_data/Plate_Maps/2022_summer_Celeste_Adaptation_16_17_18_21_sa_file.tsv'\n",
-    "if not os.path.isfile(sample_accession_fp):\n",
-    "    print(\"Problem! %s is not a path to a valid file\" % file)\n",
-    "\n",
-    "sample_accession_df = pd.read_csv(sample_accession_fp,dtype={'TubeCode':str,\n",
-    "                                                            'sample_name':str},\n",
-    "                                 sep='\\t')\n",
-    "sample_accession_df.head()"
-   ]
+   "execution_count": null
   },
   {
-   "cell_type": "markdown",
    "metadata": {},
-   "source": [
-    "## Step 2: Read in the sample info from Qiita"
-   ]
-  },
-  {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
-    "metadata_fp = './test_data/Plate_Maps/12986_20230314-090655.txt'\n",
-    "if not os.path.isfile(sample_accession_fp):\n",
-    "    print(\"Problem! %s is not a path to a valid file\" % file)\n",
-    "\n",
-    "metadata = pd.read_csv(metadata_fp, sep='\\t')\n",
-    "\n",
-    "metadata.head()"
-   ]
+    "# One dictionary per study included in the samples on this run.\n",
+    "studies_info = [\n",
+    "    # EVERY entry in the dictionary must be specifically updated \n",
+    "    # *every* time this notebook is run--none of these have defaults!\n",
+    "    {\n",
+    "    'Project Name': 'Celeste_Adaptation_12986', # PROJECTNAME_QIITAID\n",
+    "    'Project Abbreviation': 'ADAPT', # PROJECTNAME\n",
+    "    'sample_accession_fp': './test_data/Plate_Maps/sa_file_1.tsv',\n",
+    "    'qiita_metadata_fp': './test_data/Plate_Maps/12986_20230314-090655.txt',\n",
+    "    'experiment_design_description': 'isolate sequencing',\n",
+    "    'HumanFiltering': 'False', \n",
+    "    'Email': 'rodolfo.salido@gmail.com'\n",
+    "    },\n",
+    "    {\n",
+    "    'Project Name': 'CHILD_15510', # PROJECTNAME_QIITAID\n",
+    "    'Project Abbreviation': 'CHILD', # PROJECTNAME\n",
+    "    'sample_accession_fp': './test_data/Plate_Maps/sa_file_2.tsv',\n",
+    "    'qiita_metadata_fp': './test_data/Plate_Maps/15510_20240503-090339.txt',\n",
+    "    'experiment_design_description': 'whole genome sequencing',\n",
+    "    'HumanFiltering': 'True',\n",
+    "    'Email': 'lauren@ucsd.edu'\n",
+    "    },\n",
+    "    {\n",
+    "    'Project Name': 'Celeste_Marmoset_14577', # PROJECTNAME_QIITAID\n",
+    "    'Project Abbreviation': 'MARMO', # PROJECTNAME\n",
+    "    'sample_accession_fp': './test_data/Plate_Maps/sa_file_3.tsv',\n",
+    "    'qiita_metadata_fp': './test_data/Plate_Maps/14577_20230711-082202.txt',\n",
+    "    'experiment_design_description': 'whole genome sequencing',\n",
+    "    'HumanFiltering': 'False',\n",
+    "    'Email': 'celeste@ucsd.edu'\n",
```

Comment:
OK to leave this email here?

Reply:
Real emails are all over the notebooks and test files, but you have a good point--I shouldn't perpetuate that!  I will replace it with a dummy.